docs/index.html

```diff
@@ -1 +1,322 @@
+                <div data-u="slides">
+                    <div data-u="prototype" class="p" style="width:190px;height:90px;">
+                        <div data-u="thumbnailtemplate" class="t"></div>
+                        <svg viewbox="0 0 16000 16000" class="cv">
+                            <circle class="a" cx="8000" cy="8000" r="3238.1"></circle>
+                            <line class="a" x1="6190.5" y1="8000" x2="9809.5" y2="8000"></line>
+                            <line class="a" x1="8000" y1="9809.5" x2="8000" y2="6190.5"></line>
+                        </svg>
+                    </div>
+                </div>
+            </div>
+            <!-- Arrow Navigator -->
+            <div data-u="arrowleft" class="jssora106" style="width:55px;height:55px;top:162px;left:30px;" data-scale="0.75">
+                <svg viewbox="0 0 16000 16000" style="position:absolute;top:0;left:0;width:100%;height:100%;">
+                    <circle class="c" cx="8000" cy="8000" r="6260.9"></circle>
+                    <polyline class="a" points="7930.4,5495.7 5426.1,8000 7930.4,10504.3 "></polyline>
+                    <line class="a" x1="10573.9" y1="8000" x2="5426.1" y2="8000"></line>
+                </svg>
+            </div>
+            <div data-u="arrowright" class="jssora106" style="width:55px;height:55px;top:162px;right:30px;" data-scale="0.75">
+                <svg viewbox="0 0 16000 16000" style="position:absolute;top:0;left:0;width:100%;height:100%;">
+                    <circle class="c" cx="8000" cy="8000" r="6260.9"></circle>
+                    <polyline class="a" points="8069.6,5495.7 10573.9,8000 8069.6,10504.3 "></polyline>
+                    <line class="a" x1="5426.1" y1="8000" x2="10573.9" y2="8000"></line>
+                </svg>
+            </div>
+        </div>
+      </div>
+      <!-- /img slider row -->
+
+      <div class="row">
+        <div class="col-sm-4 my-4">
+          <div class="card">
+            <img class="card-img-top" src="https://raw.githubusercontent.com/RITlug/ritlug.github.io/master/img/ritlug-large.png" alt="Ritlug">
+            <div class="card-body">
+              <p class="card-text">The RIT Linux Users Group is an RIT-sponsored club for new and experienced users of the Linux operating system.</p>
+            </div>
+            <div class="card-footer">
+              <a href="http://ritlug.com/" class="btn btn-primary">RITLUG</a>
+            </div>
+          </div>
+        </div>
+        <div class="col-sm-4 my-4">
+          <div class="card">
+            <img class="card-img-top" src="img/Octocat.png" alt="">
+            <div class="card-body">
+              <h4 class="card-title">Github</h4>
+              <p class="card-text">Github is an online colaborative website where we hoste the code for Tiger-OS.</p>
```

Comment:
Please fix spelling. Also feel free to pull the description for this directly from GitHub itself